Testing/Tests/data_loading_test.py

```diff
@@ -0,0 +1,46 @@
+"""
+This class ensures that DataLoading.py correctly produces a VisIt session file.
+"""
+
+import filecmp
+import os
+import visit
+from xmldiff import main
+from PythonTool.DataLoading import py_mb_convert, plane_slice_plotting, visit_config
```

Comment:
```suggestion

from PythonTool.DataLoading import py_mb_convert, plane_slice_plotting, visit_config
```

[Pep8](https://www.python.org/dev/peps/pep-0008/#imports) recommends a blank line between groups of imports.